README.md

```diff
@@ -22,6 +23,7 @@ $ npm run start
 
 # generate static project
 $ npm run generate
+$ npm run start
```

Comment:
start is for local preview only
generate is what creates the static site.
so maybe run goes under a different heading

Reply:
Does this look OK?